core/imageroot/usr/local/agent/bin/module-backup

```diff
@@ -114,20 +137,18 @@ try:
     agent.run_restic(rdb, repository, repopath, [], ["forget", "--prune", "--keep-last=" + obackup['retention']]).check_returncode()
 
 except subprocess.CalledProcessError as ex:
-    errors = 1
+    print("[ERROR] restic backup failed.", ex, file=sys.stderr)
+    errors += 1
 
 # Advertise the backup status
-time_end = int(time.time())
-wrdb = agent.redis_connect(privileged = True)
-backup_status = {}
-stats_proc = agent.run_restic(rdb, repository, repopath, [], ["stats", "--json"], text=True, stdout=subprocess.PIPE)
+stats_proc = agent.run_restic(rdb, repository, repopath, [], ["stats", "--json", "latest"], text=True, stdout=subprocess.PIPE)
 if stats_proc.returncode == 0:
     backup_status = json.loads(stats_proc.stdout)
-    backup_status['start'] = time_start
-    backup_status['end'] = time_end
-    backup_status['errors'] = errors
-    wrdb.hset(f"module/{module_id}/backup_status/{backup_id}", mapping=backup_status)
-wrdb.close()
+else:
+    print("[ERROR] restic stats failed.", file=sys.stderr)
+    errors += 1
```

Comment:
Replace with:
    backup['errors'] += 1

Reply:
I would prefer to stick to errors if you could accept